istio-2-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1061 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local alertList = g.panel.alertList,
+      local stat = g.panel.stat,
+      local timeSeries = g.panel.timeSeries,
+      local pieChart = g.panel.pieChart,
+      local barGauge = g.panel.barGauge,
+      local table = g.panel.table,
+      local histogram = g.panel.histogram,
+
+      alertsPanel: 
+        alertList.new('Istio alerts')
+        + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+      proxies:
+        commonlib.panels.generic.stat.base.new(
+          'Proxies',
+          targets=[t.proxyCount],
+          description='Number of proxies in the Istio system.'
+        )
+        + stat.options.withGraphMode('none')
+        + stat.standardOptions.color.withMode('thresholds')
+        + stat.standardOptions.thresholds.withSteps([
+          stat.thresholdStep.withColor('super-light-red')
+          + stat.thresholdStep.withValue(null),
+          stat.thresholdStep.withColor('super-light-green')
```

Comment:
There is also a  `"unitScale": true` value that may be missing. Not sure what the default is. This applies to other panels as well.

Reply:
@JonathanWamsley I don't think I care about this one?